cmd/api/src/daemons/datapipe/datapipe.go

```diff
@@ -190,44 +187,15 @@ func (s *Daemon) Stop(ctx context.Context) error {
 }
 
 func (s *Daemon) clearOrphanedData() {
-	// Only allow one background thread to run for clearing orphaned data
-	if !s.clearOrphanedFilesLock.TryLock() {
-		return
-	}
-
-	// Release the lock once finished
-	defer s.clearOrphanedFilesLock.Unlock()
-
-	relativeTmpDir := s.cfg.TempDirectory()
-
-	if orphanFiles, err := os.ReadDir(s.cfg.TempDirectory()); err != nil {
-		log.Errorf("Failed fetching available files: %v", err)
-	} else if ingestTasks, err := s.db.GetAllIngestTasks(); err != nil {
-		log.Errorf("Failed fetching available ingest tasks: %v", err)
+	if ingestTasks, err := s.db.GetAllIngestTasks(); err != nil {
+		log.Errorf("")
```

Comment:
Is this missing error message intentional?

Reply:
NOPE

One moment.

Reply:
Patched up